Add nextcloud/coding-standard dependency

docjyJ opened the pull request:
### Add `nextcloud/coding-standard`

Attention, all files are affected!
```
Found 43 of 43 files that can be fixed in 4.645 seconds, 18.00 MB memory used
Script php-cs-fixer fix --dry-run --diff handling the cs:check event returned with error code 8
```

docjyJ commented:
I split into 2 commits:
- b3c7cbe added bibliotech
- cad1a5a run the cs

szaimen commented:
Thanks again for your contributions @docjyJ! I think this looks good but probably makes sense to wait for e.g. this PR: https://github.com/nextcloud/all-in-one/pull/4804 as otherwise it will be a rebase hell...

docjyJ commented:
Yes, I know, complicated to merge. Since it's only style, it must not be the worst case.

I dropped the commit from run cs:fix and rebased the branch.
